.github/dependabot.yml

```diff
@@ -0,0 +1,42 @@
+version: 2
+enable-beta-ecosystems: true
+updates:
+- package-ecosystem: "docker"
+  directory: "/"
+  schedule:
+    interval: weekly
+  labels:
+    - "area/dependency"
+    - "release-note-none"
+    - "ok-to-test"
+    - "kind/cleanup"
+- package-ecosystem: "docker"
+  directory: "/cmd/webhook"
+  schedule:
+    interval: weekly
+  labels:
+    - "area/dependency"
+    - "release-note-none"
+    - "ok-to-test"
+    - "kind/cleanup"
+- package-ecosystem: gomod
+  directory: "/"
+  allow:
+  - dependency-type: "all"
+  schedule:
+    interval: weekly
+  groups:
+    golang-dependencies:
+      patterns:
+        - "github.com/golang*"
+    k8s-dependencies:
+      patterns:
+        - "k8s.io*"
+        - "sigs.k8s.io*"
+        - "github.com/kubernetes-csi*"
+  labels:
+    - "area/dependency"
+    - "release-note-none"
+    - "ok-to-test"
```

Comment:
Do you also want to group github.com: https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/blob/master/go.mod#L7-L16 or it does not matter?

Reply:
Grouping all packages under github.com seems dangerous because the functionalities of the packages vary too much. However, it is a good point to group packages with similar functionalities in the same group, such as `github.com/onsi` and `github.com/spf13`. I will group some of the packages with better granularity. Please take another look at the modified groups. We can experiment and modify them on the fly.